exercises/hello-world/.version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
We use a .version in the tracks where we generate the test suites based on canonical test data defined in the github.com/exercism/x-common repository. I don't think that it's needed here (unless I'm missing something--which would not be surprising since I'm not familiar with dlang).